cli_tools/common/image/importer/api_inflater.go

```diff
@@ -292,7 +292,7 @@ func (inflater *apiInflater) getCalculateChecksumWorkflow(diskURI string) *daisy
 const (
 	checksumScriptConst = `
 		function serialOutputKeyValuePair() {
```

Comment:
This is now outputting a prefix in addtion to a key/value pair, so a different function name would be appropriate. E.g. `serialOutputPrefixedKeyValue` or similar